meterfs/meterfs.c

```diff
@@ -105,10 +105,15 @@ static int meterfs_checkfs(meterfs_ctx_t *ctx)
 
 	if (!valid0 && !valid1) {
 		LOG_INFO("meterfs: No valid filesystem detected. Formating.");
-		if ((err = ctx->partitionErase()) < 0) {
+		if ((err = ctx->eraseSector(ctx->offset + 0)) < 0) {
```

Comment:
Sorry I misslead you - header occupies more than one sector. You can use this function to safely (and easly) erase headers: `meterfs_eraseFileTable`

Reply:
Thanks for spotting this bug - changed as suggested to `meterfs_eraseFileTable()` usage.